con-base/build.gradle

```diff
@@ -2,4 +2,11 @@ jar {
     version = '0.0.1-SNAPSHOT'
     baseName = 'con-base'
     bootRepackage.enabled = false
+}
+
+dependencies {
+    compile('org.springframework.boot:spring-boot-starter-data-jpa')
+    compile('org.springframework.boot:spring-boot-starter-web')
```

Comment:
I know that Milan asked you to move spring-boot-starter-web to base module, but in my opinion this shouldn't be here. Because we also don't need web part in base module. But as this package contains also spring-core we would need it. So we have two options to solve this:
- we can move it to root gradle.build file and here exclude unneeded packages
- add only spring-core dependency here and move whole web dependency to master.